libs/core/execution/include/hpx/execution/traits/detail/simd/vector_pack_alignment_size.hpp

```diff
@@ -9,17 +9,18 @@
 
 #include <hpx/config.hpp>
 
-#if defined(HPX_HAVE_DATAPAR_STD_EXPERIMENTAL_SIMD)
+#if defined(HPX_HAVE_DATAPAR_EXPERIMENTAL_SIMD)
+
+#include <hpx/execution/traits/detail/simd/vector_pack_simd.hpp>
+
 #include <cstddef>
 #include <type_traits>
 
-#include <experimental/simd>
-
 ///////////////////////////////////////////////////////////////////////////////
 namespace hpx { namespace parallel { namespace traits {
     ///////////////////////////////////////////////////////////////////////////
     template <typename T>
-    struct is_vector_pack<std::experimental::native_simd<T>> : std::true_type
+    struct is_vector_pack<SIMD_NAMESPACE::native_simd<T>> : std::true_type
```

Comment:
Instead of this macro, could we do something similar as done for `hpx::variant` here: https://github.com/STEllAR-GROUP/hpx/blob/master/libs/core/datastructures/include/hpx/datastructures/variant.hpp? I.e. introduce `hpx::experimental::native_simd` that is an alias for the imported type from either `namespace sve::experimental` or `namespace std::experimental`.